### ISSUE TITLE
Freeze when processing `color-mix()`

### ISSUE THREAD

**Issue by romainmenke** [parcel-bundler/lightningcss#446]
https://lightningcss.dev/playground/index.html#%7B%22minify%22%3Afalse%2C%22nesting%22%3Atrue%2C%22customMedia%22%3Atrue%2C%22cssModules%22%3Afalse%2C%22analyzeDependencies%22%3Afalse%2C%22targets%22%3A%7B%22chrome%22%3A6225920%7D%2C%22source%22%3A%22%23e%20%7B%5Cn%20%20background-color%3A%20color-mix(in%20xyz%2C%20transparent%2C%20green%2065%25)%3B%5Cn%7D%22%2C%22visitorEnabled%22%3Afalse%2C%22visitor%22%3A%22%7B%5Cn%20%20Color(color)%20%7B%5Cn%20%20%20%20if%20(color.type%20%3D%3D%3D%20'rgb')%20%7B%5Cn%20%20%20%20%20%20color.g%20%3D%200%3B%5Cn%20%20%20%20%20%20return%20color%3B%5Cn%20%20%20%20%7D%5Cn%20%20%7D%5Cn%7D%22%2C%22unusedSymbols%22%3A%5B%5D%2C%22version%22%3A%22local%22%7D

```css
#e {
  background-color: color-mix(in xyz, transparent, green 65%);
}
```

This freezes the playground

**Comment by devongovett** [comment]
Good catch! Looks like this was due to rounding errors in the gamut mapping algorithm. The spec was updated at some point to use an epsilon check instead of a strict < comparison.

Btw, looks like there were some other changes in https://github.com/w3c/csswg-drafts/commit/e30aec4ca7c827356a3091008117c662bd7e0fe2 but seems like WPT hasn't been updated yet and there were a lot of test changes when I updated Lightning CSS so I bailed for now. Asked about updating WPT in https://github.com/w3c/csswg-drafts/issues/7135#issuecomment-1501172138
